src/app/detail/[task_id]/[user_type]/page.tsx

```diff
@@ -67,7 +68,7 @@ export default async function TaskDetailPage({
   const [task, tokenPayload] = await Promise.all([getOneTask(token, task_id), copilotClient.getTokenPayload()])
   // Basic validation
   if (!tokenPayload) {
-    throw new Error('Token cannot be found')
+    return <SilentError message="Please provide a Valid Token" />
```

Comment:
Please help me understand what is the purpose of building `SilentError`? Why can't the general `ClientErrorBoundary` would just work fine?

Reply:
@aatbip for reusability. This same logic is used in all 3 of our pages. I don't want to have to pass `new Error("Please provide a valid token" and reset on all 3 pages.

Reply:
Is the question why I made a separate component for this, or why i'm not throwing an error to render ClientErrorBoundary?

Reply:
First comment answered my question. Thank you for the clarification.